BUILDING.md

```diff
@@ -74,28 +76,30 @@ $ ninja -C build install
 ```
 
 The `install` step will install rizin in `~/.local/bin`, so make sure to add it
-to your `PATH` variable. As most systems don't look for libraries in
-`~/.local/lib`/`~/.local/lib64`, you will have to set `LD_LIBRARY_PATH`
-accordingly or, if you prefer, use `meson -Dlocal=true --prefix=~/.local build`
-instead of just `meson --prefix=~/.local build`.
+to your `PATH` variable (e.g. `export PATH=$PATH:~/.local/bin`).
+
+NOTE: meson will set `RPATH` to ensure that libraries can be found on the
```

Comment:
```suggestion
NOTE TO DISTRIBUTION PACKAGERS: meson will set `RPATH` to ensure that libraries can be found on the
```

This is just a suggestion.

Reply:
I would personally leave it for everyone, it's not really just for distro packegers. Is it ok if i leave it as-is?

Reply:
I think so